hack/make.sh

```diff
@@ -118,8 +118,10 @@ fi
 
 if [ -z "$DOCKER_CLIENTONLY" ]; then
 	DOCKER_BUILDTAGS+=" daemon"
-	if pkg-config libsystemd-journal 2> /dev/null ; then
+	if pkg-config 'libsystemd >= 209' 2> /dev/null ; then
```

Comment:
When I try it at the command line, pkg-config thinks I'm talking about a package named "||", and the compiler gives me a "malformed #cgo argument: >=" error when I go ahead and try it in the source file.